diff --git a/classes/submission/reviewer/form/PKPReviewerReviewStep3Form.php b/classes/submission/reviewer/form/PKPReviewerReviewStep3Form.php
index 9a283b5b1..fba7a6ae4 100644
--- a/classes/submission/reviewer/form/PKPReviewerReviewStep3Form.php
+++ b/classes/submission/reviewer/form/PKPReviewerReviewStep3Form.php
@@ -79,6 +79,8 @@ class PKPReviewerReviewStep3Form extends ReviewerReviewForm
         $submissionCommentsPrivate = $submissionCommentDao->getReviewerCommentsByReviewerId($reviewAssignment->getSubmissionId(), $reviewAssignment->getReviewerId(), $reviewAssignment->getId(), false);
         $submissionCommentPrivate = $submissionCommentsPrivate->next();
         $this->setData('commentsPrivate', $submissionCommentPrivate ? $submissionCommentPrivate->getComments() : '');
+
+	parent::initData();
     }
 
     //
